src/ray/raylet/node_manager.h

```diff
@@ -110,7 +112,8 @@ struct NodeManagerConfig {
   int64_t min_spilling_size;
 };
 
-class NodeManager : public rpc::NodeManagerServiceHandler {
+class NodeManager : public rpc::NodeManagerServiceHandler,
```

Comment:
Can you also remove the new_scheduler_enabled_ member (I think it's only needed in the constructor now)?